specification/gedcom-3-structures-4-enumerations.md

```diff
@@ -230,14 +230,83 @@ and applications should be prepared to encounter non-current values.
+| `HONORIFIC` | A word or phrase attached to a name in formal or polite context to indicate station, such as "Miss", "Doctor", "さん", "様", "mademoiselle", and so on. |
+| `IMMIGRANT` | Name assumed at the time of immigration. |
+| `INFORMAL` | A name only used in casual, intimate, or informal settings. |
+| `LEGAL` | A name used for legal and official documents, but not in daily use. |
+| `LOCATION` | A name indicating a location of note, such as a city associated with the person. Often includes "of" or "from" type particles. Incompatible with `SURN`. |
+| `MAIDEN` | Maiden name, name before first marriage. |
+| `MARRIED` | Married name, assumed as part of marriage. |
+| `MATERNAL` | A name inherited from the individuals' mother's family. Implies `SURN`. |
+| `MATRONYMIC` | A name of the individual's mother, possibly with a matronymic modifier. |
+| `NICK` | A descriptive or familiar name that is used instead of, or in addition to, one’s official or legal name. Some cultures use this for any name that is not used in legal documents, others only for names that would be inappropriate in formal settings. |
+| `NPFX` | Text that appears on a name line before the given and surname parts of a name. Implies that the person attaches this part to their name, but does not consider it part of the name itself. |
+| `NSFX` | Text which appears on a name line after or behind the given and surname parts of a name. Implies that the person attaches this part to their name, but does not consider it part of the name itself. |
+| `PARTICLE` | A name part that connects or modifies other name parts but is not itself considered a name, like "of" or "son of". | 
+| `PATERNAL` | A name inherited from the individuals' father's family. Implies `SURN`. |
+| `PATRONYMIC` | A name of the individual's father, possibly with a patronymic modifier like prefix "bar" or "di ser" or suffix "sen" or "dotter". |
+| `PRIMARY` | The name of most prominent in importance among the names of that type. Requires `GIVN`, `SURN`, `NPFX`, or `NSFX`. |
+| `PROFESSIONAL` | Name used professionally (pen, screen, stage name). |
+| `RANK` | A designation of rank or position, for example in a military ("private first class"), nobility ("viscount de Spoelberch"), or  educational ("Ph.D.") system. |
+| `RELIGIOUS` | Religious name, name adopted when joining a religious order. |
+| `ROEPNAAM` | A name provided at birth for use in all situations except legal documents. Implies `GIVN` and `BIRTH`. The tag of this value comes from Dutch instead of English because no suitable English word was found; the value does not imply Dutch culture or ancestry. |
+| `RUFNAME` | A given name underlined or otherwise indicated on documents as one not to be omitted when only one given name is used. Implies `GIVN` and `PRIMARY`. The tag of this value comes from German instead of English because no suitable English word was found; the value does not imply German culture or ancestry. |
```

Comment:
The "only one" part of this definition is confusing/ambiguous.  Does that mean it is only permitted to have one `RUFNAME` per `NAME`?  (There may be some interaction with the `PRIMARY` discussion as well.)